android/build.gradle

```diff
@@ -54,12 +54,12 @@ android {
     compileSdkVersion safeExtGet("compileSdkVersion", 31)
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+        // sourceCompatibility JavaVersion.VERSION_11
+        // targetCompatibility JavaVersion.VERSION_11
     }
 
     kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_11.majorVersion
+        // jvmTarget = JavaVersion.VERSION_11.majorVersion
```

Comment:
I'm wondering what's the risk of removing the `compileOptions` and `kotlinOptions` outright. I haven't tested that out but it seems like that's the right move here, instead of commenting out or removing these changed lines.